lib/views/issueish-list-view.js

```diff
@@ -0,0 +1,55 @@
+import React, {Fragment} from 'react';
+import PropTypes from 'prop-types';
+
+import {SearchPropType} from '../prop-types';
+import {autobind} from '../helpers';
+import Accordion from './accordion';
+import Octicon from '../atom/octicon';
+
+export default class IssueishListView extends React.Component {
+  static propTypes = {
+    search: SearchPropType.isRequired,
+    isLoading: PropTypes.bool.isRequired,
+    total: PropTypes.number.isRequired,
+    issueishes: PropTypes.arrayOf(PropTypes.any).isRequired,
+  }
+
+  constructor(props) {
+    super(props);
+
+    autobind(this, 'renderIssueish');
+  }
+
+  render() {
+    return (
+      <Accordion
+        leftTitle={this.props.search.getName()}
+        isLoading={this.props.isLoading}
+        results={this.props.issueishes}>
+        {this.renderIssueish}
+      </Accordion>
+    );
+  }
+
+  renderIssueish(issueish) {
+    return (
+      <Fragment>
+        <img
+          className="github-IssueishList-item--avatar"
+          src={issueish.getAuthorAvatarURL() + "&s=32"}
```

Comment:
Ah right cool. I've added a `size` parameter to the `Issueish.getAuthorAvatarURL()` method and used `URL` manipulation methods to add the query string parameter... just in case.